### PR TITLE
repositories: add heroku-miraheze

### PR DESCRIPTION
**Pull request by heroku-miraheze**

### CODE REVIEW

**Comment by jsmolic** [comment]
Thanks for the PR, there are few requirements [1] all overlays need to have before we can add them (such as having `profiles/repo_name` and `masters` entry in `metadata/layout.conf`)

[1] https://wiki.gentoo.org/wiki/Project:Overlays/Overlays_guide#Basic_rules

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5054,5 +5054,18 @@
     <source type="git">git@github.com:kabili207/zyrenth-overlay.git</source>
     <feed>https://github.com/kabili207/zyrenth-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>heroku-miraheze</name>
```
Comment (jsmolic) [inline files/overlays/repositories.xml:5058]: Please keep the list sorted as well

**Review by jsmolic — CHANGES_REQUESTED** [review]

File: files/overlays/repositories.xml (hunk 1 above)
Comment (heroku-miraheze) [reply files/overlays/repositories.xml]: Fixed

**Comment by heroku-miraheze** [comment]
@jsmolic Fixed, now, you can merge it now.

**Comment by thesamesam** [comment]
I'm sorry, your attitude on IRC isn't something we're able to work with. It is not appropriate to ping every member of a project repeatedly within a few minutes and then PM several members demanding a merge.

**Comment by heroku-miraheze** [comment]
I cooled off, so, reopen my PR. @thesamesam

**Pull request closed without merging**